app/models/concerns/orchestration.rb

```diff
@@ -82,14 +90,26 @@ def record_conflicts
     @record_conflicts ||= []
   end
 
+  def skip_orchestration!
+    @skip_orchestration = true
+  end
+
+  def enable_orchestration!
+    @skip_orchestration = false
+  end
+
+  def skip_orchestration?
+    @skip_orchestration ||= Rails.env == "test"
```

Comment:
Could you rewrite this as:

    return true if Rails.env.test?
    !!@skip_orchestration

or similar, perhaps with a comment?  I think the intention will be far clearer that way, and as you say, this was the original code extracted from line 92.  Currently the intention is less than clear because `Rails.env` is only tested when skip_orchestration is already false.